### ISSUE TITLE
Recommendations for citing

### ISSUE THREAD

**Issue by emdupre** [bids-standard/bids-matlab#20]
Thanks for making this resource available !

It would be great to be able to cite it directly. I'm not sure if you're yet ready to version this and [create a DOI](https://guides.github.com/activities/citable-code/), but I wonder if [generating an RRID](https://scicrunch.org/resources/about/resource?resource_suggestion) would be of interest. Happy to help, but I wanted to check in with you, first !

**Comment by Remi-Gau** [comment]
I don't think we have gone there yet but I can take care of the DOI via zenodo. 
@gllmflndn: should I take care of it?
Never done RRID but I can have a look

**Comment by gllmflndn** [comment]
Many thanks @emdupre for the links. We should definitely create a DOI when we start making releases - it might take a bit of time as we should first set the API in stone (see  #19) so let's keep this issue open until then as a reminder.
@Remi-Gau: the RRID form has a "Defining Citation" field so best to wait for the DOI first - unless I misunderstand something?

**Comment by Remi-Gau** [comment]
> @Remi-Gau: the RRID form has a "Defining Citation" field so best to wait for the DOI first - unless I misunderstand something?

Noted

**Comment by emdupre** [comment]
Thanks for checking in on this, @gllmflndn and @Remi-Gau !

Sorry for being unclear on my part -- that "Defining Citation" field is optional, so I wouldn't expect you to have one. For a bit more context, I've seen RRIDs used interchangeably with the "concept DOI" that Zenodo will mint for your project once you've started versioning releases. From [the original RRID paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4648211/):

> A case could be made for using DOIs to identify particular software tools and databases, as they are digital objects. As discussed in the next section, our preference is that DOIs be used to identify the particular instance used, e.g., the version of data or software and any supporting workflows, and that the RRID be used to identify the entity or project referenced.

You can find an example usage in [the fMRIPrep citation boilerplate](https://github.com/poldracklab/fmriprep/blob/278fb84c5407520cdbcc7228c8bcbe8f82d19c6c/docs/citing.rst).

All that to say, I think you _can_ create an RRID now, if that's of interest. But if not I'll just direct folks to link to specific commits or include access dates in their references.

**Comment by Remi-Gau** [comment]
Tempted to look into the RRID thing again now.

Also anyone knows why I can't "see" this bids-matlab repo on zenodo (was looking into the versioning issue).

I can see repos of some other organizations I am part of but not this one. I suspect it is a "access rights" issue.

**Comment by gllmflndn** [comment]
> Also anyone knows why I can't "see" this bids-matlab repo on zenodo (was looking into the versioning issue).

And now? (with great powers etc etc)

**Comment by Remi-Gau** [comment]
> And now? 

Not yet. No sure how long it takes to show up.

> (with great powers etc etc)

comes the ability to detect smaller effect sizes?

**Comment by gllmflndn** [comment]
I don't know then. You are now an admin of the bids-matlab repository so perhaps you need to be higher up in the hierarchy of the bids-standard organisation.

And trying hard to compete with your power joke but I think you won :rofl:

**Comment by sappelhoff** [comment]
I *can* see it on zenodo (screenshot):

![image](https://user-images.githubusercontent.com/9084751/97141337-d1d9ab00-175e-11eb-8def-2765105f49f7.png)

Remi is admin for bids-matlab (screenshot):

![image](https://user-images.githubusercontent.com/9084751/97141354-dc944000-175e-11eb-8b10-c2edd4614f16.png)


And on the bids-standard gitHub org, Remi is "member". However there are only "members" and "admins", nobody in between, so I don't know why Remi does *not* see the zenodo switch.Screenshot:

![image](https://user-images.githubusercontent.com/9084751/97141470-0f3e3880-175f-11eb-9bd2-e5fe1a19dddc.png)

Are you logged in on Zenodo "via GitHub" @Remi-Gau ?

**Comment by Remi-Gau** [comment]
Ha... I had not checked in a while.

It is there now! I activated it so we should get everything synched on zenodo when we release. :-)

**Comment by Remi-Gau** [comment]
FYI: there a CITATION.cff file in the `dev` branch that was added by #284.

After #287 is merged in master I will make a first release that should hopefully be picked by zenodo and give us a DOI.
